en-US/focus-ios.xliff

```diff
@@ -201,6 +206,11 @@
         <target>Browse like no one’s watching.</target>
         <note>Message label on the first run screen</note>
       </trans-unit>
+      <trans-unit id="Focus.handoffSyncing">
+        <source>Apple Handoff is syncing</source>
+        <target>Apple Handoff is syncing</target>
+        <note>Title for the loading screen when the handoff of clipboard delays Focus launch.</note>
```

Comment:
Looks like Handoff is a name that should not be localized
https://support.apple.com/it-it/HT204681
https://support.apple.com/es-es/HT204681

We should add that into the localization note: "Title for the loading screen when the handoff of clipboard delays Focus launch. Handoff should not be localized, see https://support.apple.com/HT204681".